### ISSUE TITLE
Listed repo is not recognized

### ISSUE THREAD

**Issue by gpkvt** [conceptsandtraining/doil#165]
```
doil repo:list
Currently registered private repositories:
/usr/local/lib/doil/lib/repo/list/list.sh: line 44: ~/.doil/config/repositories.conf: No such file or directory

Currently registered global repositories:
        ilias - git@github.com:ILIAS-eLearning/ILIAS.git
```

but:

```
doil instances:create -n testing -r ilias -b release_7 -p 7.3
sed: can't read ~/.doil/config/repositories.conf: No such file or directory
ERROR:
        Repository ilias does not exist!
        Use doil repo:list to see all repositories!
```

**Comment by gpkvt** [comment]
Oh, I should have used `-gr` instead of `-r`. Is this separation really needed?

**Comment by gpkvt** [comment]
doil need to know this, but does the user need to know it as well? If there are a private and and public repo named ilias, ok, then doil need to know which one is meant. But if there is only one repo with this name, it may just use it.

**Comment by lauraherzog** [comment]
It's an ux problem. So no fix here, but I took a note for future discussion.

**Comment by lauraherzog** [comment]
Will be fixed with #168
